src/Core/System/Snippet/Command/ValidateSnippetsCommand.php

```diff
@@ -27,20 +27,26 @@ class ValidateSnippetsCommand extends Command
      * @internal
      */
     public function __construct(
-        private readonly SnippetValidatorInterface $snippetValidator,
+        private readonly BundleSnippetValidatorInterface $snippetValidator,
         private readonly SnippetFixer $snippetFixer
     ) {
         parent::__construct();
     }
 
     protected function configure(): void
     {
-        $this->addOption('fix', 'f', InputOption::VALUE_NONE, 'Use this option to start a wizard to fix the snippets comfortably');
+        $this
+            ->addOption('fix', 'f', InputOption::VALUE_NONE, 'Use this option to start a wizard to fix the snippets comfortably')
+            ->addOption('bundle', 'b', InputOption::VALUE_OPTIONAL, 'The bundle name where to load the snippets. Multiple bundles can be passed separated by a comma', 'Storefront, Administration')
+        ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $missingSnippetsArray = $this->snippetValidator->validate();
+        /** @var list<string> $bundles */
+        $bundles = array_map('trim', array_filter(explode(',', (string) $input->getOption('bundle'))));
```

Comment:
```suggestion
        $bundles = array_values(array_map('trim', array_filter(explode(',', (string) $input->getOption('bundle')))));
```

then you do not need the var annotation anymore